modular_skyrat/modules/colony_fabricator/code/design_datums/tools.dm

```diff
@@ -30,9 +31,22 @@
 		RND_CATEGORY_TOOLS + RND_SUBCATEGORY_TOOLS_ENGINEERING_ADVANCED,
 	)
 
+/datum/design/colony_door_crowbar_replacement
+	name = "Crowbar"
+	id = "colony_crowbar"
+	build_type = COLONY_FABRICATOR
+	build_path = /obj/item/crowbar/red
+	materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT *0.5,
+	)
+	category = list(
+		RND_CATEGORY_INITIAL,
+		RND_CATEGORY_TOOLS + RND_SUBCATEGORY_TOOLS_ENGINEERING_ADVANCED,
+	)
+
```

Comment:
Oh I see, it's just giving a red crowbar as a replacement. It would honestly be better if you created a file inside modular_zubbers specifically to add overrides and the new design path.

Reply:
Basically you can copy all this code into an entirely new .dm and tick it, and it'll add this design to the fabricator. Then when you remove it, you can just delete the file.